Todo/Todo.cs

```diff
@@ -0,0 +1,627 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Globalization;
+using System.Security.Permissions;
+using System.Threading;
+
+namespace To_Do_List
+{
+    public class To_do : IComparable<To_do>
+    {
+        public int Todo_ID { get; set; }
+        public DateTime date { get; set; }
+        public string Task { get; set; }
+        public int Lvl_Imp { get; set; }
+
+        public To_do(int Todo_ID, DateTime date, string Task, int Lvl_Imp)
+        {
+            this.Todo_ID = Todo_ID;
+            this.date = date;
+            this.Task = Task;
+            this.Lvl_Imp = Lvl_Imp;
+        }
+        public int CompareTo(To_do other)
+        {
+            return this.Todo_ID.CompareTo(other.Todo_ID);
+        }
+
+    }
+
+
+    class Program
+    {
```

Comment:
While your program functioned and did allow me to add an item, it is mostly all in 1 class without any really separation between classes and functionality. This defeats the purpose of working with an object oriented language like C#.